src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -423,29 +416,28 @@ function updateDOMProperties(
       }
     } else if (isCustomComponentTag) {
       DOMPropertyOperations.setValueForAttribute(
-        getNode(workInProgress),
+        domElement,
         propKey,
         nextProp
       );
     } else if (
         DOMProperty.properties[propKey] ||
         DOMProperty.isCustomAttribute(propKey)) {
-      var node = getNode(workInProgress);
       // If we're updating to null or undefined, we should remove the property
       // from the DOM node instead of inadvertently setting to a string. This
       // brings us in line with the same behavior we have on initial render.
       if (nextProp != null) {
-        DOMPropertyOperations.setValueForProperty(node, propKey, nextProp);
+        DOMPropertyOperations.setValueForProperty(domElement, propKey, nextProp);
       } else {
-        DOMPropertyOperations.deleteValueForProperty(node, propKey);
+        DOMPropertyOperations.deleteValueForProperty(domElement, propKey);
       }
     }
   }
   if (styleUpdates) {
     CSSPropertyOperations.setValueForStyles(
-      getNode(workInProgress),
+      domElement,
       styleUpdates,
-      workInProgress
+      null // TODO: Change CSSPropertyOperations to use getCurrentOwnerName.
```

Comment:
so we are planning to consistently set owner during reconciliation instead of looking things up by internal debug ID?